scipy/stats/_continuous_distns.py

```diff
@@ -3364,9 +3364,44 @@ class levy_stable_gen(rv_continuous):
 
     Notes
     -----
-    Levy-stable distribution (only random variates available -- ignore other
-    docs)
 
+    The distribution for `levy_stable` is parametrized by
+    :math:`0 < a \leq 2` and :math:`-1 \leq b \leq 1`.  Its probability
+    density function cannot be expressed analytically, but is described by
+    the Fourier transform of its characteristic function [1]_
+
+    .. math::
+
+        f(x) &= \frac{1}{2\pi}\int_{-\infty}^\infty \varphi(t)e^{-ixt}\,dt \\
+        \varphi(t;a,b) &= \exp\left (it - |t|^a
```

Comment:
Since the characteristic function _is_ the Fourier transform of the PDF, the Fourier inverse of the characterisitc function is the PDF (for any continuous distribution). I would simply specify the characteristic function which fully specifies the distribution.
in your definition of \varphi: is the "it" correct? (on Wikipedia, it says "\mu i t", but \mu is zero here, no? I might be wrong...)

Reply:
Ah, you're right, I switched the scale & shape parameters.